test/resources/liftover_lb_lp.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#SBATCH --job-name=vip_test_liftover
+#SBATCH --output=vip_test_liftover.out
+#SBATCH --error=vip_test_liftover.err
+#SBATCH --time=01:00:00
+#SBATCH --cpus-per-task=4
+#SBATCH --mem=16gb
+#SBATCH --nodes=1
+#SBATCH --export=NONE
+#SBATCH --get-user-env=L60
+#SBATCH --tmp=4gb
+
+ml picard
```

Comment:
How about creating a singularity container for picard and using it here? Lets try not to reintroduce EasyBuild.